funnel/assets/js/project.js

```diff
@@ -53,13 +53,59 @@ const TicketWidget = {
       });
     }, false);
 
+    this.initTicketModal();
     this.trackBoxofficeEvents();
   },
   trackBoxofficeEvents() {
     $(document).on('boxofficeTicketingEvents', (event, userAction, label, value) => {
       Utils.sendToGA('ticketing', userAction, label, value);
     });
   },
+  initTicketModal() {
+    $('.open-ticket-widget').click(event => {
+      event.preventDefault();
+      this.openTicketModal();
+    });
+
+    $('#close-ticket-widget').click(event => {
+      event.preventDefault();
+      this.hideTicketModal();
+    });
+
+    if (window.location.hash.indexOf('#tickets') > -1) {
+      if ($(window).width() < 768) {
+        this.openTicketModal();
+      } else {
+        Utils.animateScrollTo($('#tickets').offset().top);
+      }
+    }
+
+    $(window).resize(() => {
+      if ($(window).width() > 767 && $('.about__participate').hasClass('about__participate--modal')) {
```

Comment:
Can this 767 be moved to global config? We have to periodically revise screen sizes based on current devices, so having them centralised will help.